js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -129,7 +150,34 @@ export default class AddressSelect extends Component {
     };
   }
 
+  renderBalance (address) {
+    const { balances = {} } = this.props;
+    const balance = balances[address];
+
+    if (!balance) {
+      return null;
+    }
+
+    const ethToken = balance.tokens.find((t) => t.token.tag.toLowerCase() === 'eth');
```

Comment:
1. Naming is different from the `tok.token` in balancesReducer.
2. No nulls here?